server/routes/index.js

```diff
@@ -2,7 +2,11 @@ const express = require("express");
 const router = express.Router();
 
 router.get("/welcome", function (req, res, next) {
-  res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
+	res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
```

Comment:
You can remove these placeholder routes now that you're creating some real routes :)